service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -41,7 +41,8 @@ private RunLog runToRunLog(Run run) {
         .state(CbasRunStatus.toCbasApiState(run.status()))
         .workflowParams(run.runSet().method().inputDefinition())
         .submissionDate(convertToDate(run.submissionTimestamp()))
-        .lastModifiedTimestamp(convertToDate(run.lastModifiedTimestamp()));
+        .lastModifiedTimestamp(convertToDate(run.lastModifiedTimestamp()))
+        .errorMessages(null);
```

Comment:
You're already adding this to the `Run` class and fetching from the database in `RunDao`... why not pass it through here (eg `.errorMessages(run.errorMessages())`?)